src/main/java/org/swmaestro/repl/gifthub/auth/controller/AuthController.java

```diff
@@ -100,36 +78,33 @@ public ResponseEntity<Message> reissueAccessToken(HttpServletRequest request, @R
 		String newAccessToken = refreshTokenService.createNewAccessTokenByValidateRefreshToken(refreshToken);
 		String newRefreshToken = refreshTokenService.createNewRefreshTokenByValidateRefreshToken(refreshToken);
 
-		TokenDto tokenDto = TokenDto.builder()
+		JwtTokenDto jwtTokenDto = JwtTokenDto.builder()
 				.accessToken(newAccessToken)
 				.refreshToken(newRefreshToken)
 				.build();
 
 		refreshToken = refreshToken.substring(7);
-		refreshTokenService.storeRefreshToken(tokenDto, jwtProvider.getUsername(refreshToken));
+		refreshTokenService.storeRefreshToken(jwtTokenDto, jwtProvider.getUsername(refreshToken));
 
 		return ResponseEntity.ok(
 				SuccessMessage.builder()
 						.path(request.getRequestURI())
-						.data(tokenDto)
+						.data(jwtTokenDto)
 						.build());
 	}
 
 	@PostMapping("/sign-in/naver")
-	@Operation(summary = "네이버 로그인 콜백 메서드", description = "네이버 로그인 콜백을 하기 위한 메서드입니다.")
+	@Operation(summary = "네이버 로그인 메서드", description = "네이버 로그인 콜백을 하기 위한 메서드입니다.")
 	@ApiResponses({
 			@ApiResponse(responseCode = "200", description = "네이버 로그인 성공"),
 			@ApiResponse(responseCode = "400", description = "네이버 로그인 실패"),
 	})
-	public ResponseEntity<Message> naverSignIn(HttpServletRequest request, @RequestBody TokenDto token) throws IOException {
-		NaverDto naverDto = naverService.getUserInfo(token);
-		Member member = naverService.signUp(naverDto);
-		oAuthService.save(member, OAuthPlatform.NAVER, naverDto.getId());
-		TokenDto tokenDto = naverService.signIn(naverDto, member.getId());
+	public ResponseEntity<Message> naverSignIn(HttpServletRequest request, @RequestBody OAuthTokenDto oAuthTokenDto) {
+		JwtTokenDto jwtTokenDto = authService.signIn(oAuthTokenDto, OAuthPlatform.KAKAO);
```

Comment:
p1:
`OAuthPlatform.KAKAO` -> 오타 있습니다! 
```suggestion
		JwtTokenDto jwtTokenDto = authService.signIn(oAuthTokenDto, OAuthPlatform.NAVER);
```